udata/core/followers/api.py

```diff
@@ -55,7 +55,7 @@ def get(self, id):
     @api.doc('follow', description=NOTE)
     def post(self, id):
         '''Follow an object given its ID'''
-        model = self.model.objects.only('id').get_or_404(id=id)
+        model = self.model.objects.get_or_404(id=id)
```

Comment:
OK, I just pushed a proposal for the 'long term' fix ;-)

Still, it does not explain why the test does not crash without the fix.